cmd/drone-acr/main.go

```diff
@@ -1,17 +1,46 @@
 package main
 
 import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
 	"os"
 	"os/exec"
 	"strings"
 
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
 	"github.com/joho/godotenv"
+	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 
 	docker "github.com/drone-plugins/drone-docker"
 )
 
+type subscriptionUrlResponse struct {
+	Value []struct {
+		ID string `json:"id"`
+	} `json:"value"`
+}
+
+const (
+	acrCertPath              = "/tmp/acr-cert.pem"
```

Comment:
Does this work for Windows?

Reply:
good catch! I'll change it to use `os.TempDir()` to keep it more generic and cross platform